.circleci/create_circleci_config.py

```diff
@@ -99,10 +103,57 @@ def to_dict(self):
             f"--make-reports={self.name}" if "examples" in self.name else f"--make-reports=tests_{self.name}"
         )
         test_command = f"python -m pytest -n {self.pytest_num_workers} " + " ".join(pytest_flags)
-        if self.tests_to_run is None:
-            test_command += " << pipeline.parameters.tests_to_run >>"
+        if self.parallelism == 1:
```

Comment:
Keep the original logic for `parallelism=1`, as @sgugger feels the the logic general case is a bit too complex.